data/nmstate/001-rbac

```diff
@@ -0,0 +1,64 @@
+{{ if .EnableSCC }}
```

Comment:
We need the RBAC regardless if scc is enable.

but we need to add a section of scc 

you can take a look on this yaml https://github.com/kubevirt/cluster-network-addons-operator/blob/master/data/sriov/001-rbac.yaml

something like this
```
{{ if .EnableSCC }}
---
apiVersion: security.openshift.io/v1
kind: SecurityContextConstraints
metadata:
  name: nmstate
allowPrivilegedContainer: true
allowHostDirVolumePlugin: true
runAsUser:
  type: RunAsAny
seLinuxContext:
  type: RunAsAny
users:
- system:serviceaccount:nmstate:nmstate-controller
{{ end }}
```

Reply:
ok, understood.  SCC is an openshift construct that one can use to control how things are being executed (in this case as a service account)